go/client/passphrase_prompt.go

```diff
@@ -16,6 +16,7 @@ func PromptPassphrase(g *libkb.GlobalContext) (keybase1.GetPassphraseRes, error)
 	arg := libkb.DefaultPassphraseArg()
 	arg.WindowTitle = "Passphrase"
 	arg.Prompt = "Pick a strong passphrase (12+ characters)"
+	arg.Type = keybase1.PassphraseType_PASS_PHRASE
```

Comment:
@patrickxb we're plumbing through the 'type' of the passphrase request. this is cause we show different screens (outside of the pinentry) when you're provisioning through the gui